coderd/database/pubsub/pubsub.go

```diff
@@ -262,7 +276,11 @@ func (p *pgPubsub) closeListener() error {
 // listen begins receiving messages on the pq listener.
 func (p *pgPubsub) listen() {
 	defer func() {
-		_ = p.closeListener()
+		p.logger.Info(p.ctx, "pubsub stopped receiving notify")
+		cErr := p.closeListener()
+		if cErr != nil {
+			p.logger.Error(p.ctx, "failed to close listener")
+		}
```

Comment:
A bit below there's the nil check for "reconnect" behavior, should we log there as well (just to see if it happens often)?

Reply:
I think logging that would just dupe the "pubsub reconnected to postgres" log, so it doesn't add any new value.

Reply:
I guess I'm thinking about potential error scenarios we can't foresee, like what if we keep getting nil messages after a reconnect cycle?

Reply:
I could drop a debug message, I guess...